ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,154 @@
+    ...                                    tol_seconds=${ACC_TOLERATION_SECONDS}
+    In The Accelerator Profiles Grid There Is An Accelerator Profile With Name   ${ACC_DISPLAY_NAME}01
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}01   ${ACC_IDENTIFIER}
+    ...                                         ${ACC_DESCRIPTION}   ${ACC_ENABLED}    tolerations=yes
+    ...                                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                                         tol_key=${ACC_TOLERATION_KEY}
+    ...                                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                                         tol_seconds=${ACC_TOLERATION_SECONDS}
+
+Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI
+    [Documentation]    Modify an Accelerator Profile instance from the Administration UI and verify it's content
+    [Tags]  ODS-XXX
+    ...     Smoke
+    [Setup]  Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}2
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Click On Edit Accelerator Profile    ${ACC_DISPLAY_NAME}2
+    Modify The Accelerator Profile    original_display_name=${ACC_DISPLAY_NAME}2
+    ...                               display_name=${ACC_DISPLAY_NAME}2_modified
+    ...                               identifier=${ACC_IDENTIFIER}_modified
+    ...                               description=${ACC_DESCRIPTION}_modified    tolerations=yes
+    ...                               tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                               tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                               tol_key=${ACC_TOLERATION_KEY}
+    ...                               tol_value=${ACC_TOLERATION_VALUE}
+    ...                               tol_seconds=${ACC_TOLERATION_SECONDS}
+    In The Accelerator Profiles Grid There Is An Accelerator Profile With Name   ${ACC_DISPLAY_NAME}2_modified
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}2_modified   ${ACC_IDENTIFIER}_modified
+    ...                                         ${ACC_DESCRIPTION}_modified   tolerations=yes
+    ...                                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                                         tol_key=${ACC_TOLERATION_KEY}
+    ...                                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                                         tol_seconds=${ACC_TOLERATION_SECONDS}
+
+Delete Tolerations from an Accelerator Profile Using "Accelerator Profiles" Administration UI
+    [Documentation]    Delete Tolerations from  an Accelerator Profile instance from the Administration UI and
+    ...                verify it's content
+    [Tags]  ODS-XXX
```

Comment:
What's the plan with all these `ODS-XXX` references?

Reply:
I left that tag waiting to speak with @manosnoam about which one should I use. Fixed in 
a951d55